source/TimePicker.js

```diff
@@ -419,9 +414,31 @@
 					fmtParams.locale = this.locale;
 				}
 				var merFormatter = new ilib.DateFmt(fmtParams);
-				var am = ilib.Date.newInstance({hour:10});
-				var pm = ilib.Date.newInstance({hour:14});
-				this.meridiems = [merFormatter.format(am), merFormatter.format(pm)];
+				
+				this.meridiems = merFormatter.getMeridiemsRange(fmtParams);
```

Comment:
Thank you. Let me update it from 2.6.0-dev.
Cause I'd like to make at least changes as I can to prevent regression issue.